Makefile

```diff
@@ -62,9 +62,9 @@ gen-k8s: $(apis_sources)
 gen-k8s-check: $(apis_sources)
 	./hack/verify-codegen.sh
 
-container-build: container-build-operator container-build-registry
+container-build: csv-generator container-build-operator container-build-registry
```

Comment:
you might be able to remove csv-generator here since `container-build-operator:` use it